src/main/java/com/cruru/process/service/ProcessService.java

```diff
@@ -0,0 +1,119 @@
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ProcessService {
+
+    private static final int MAX_PROCESS_COUNT = 5;
+    private static final int PROCESS_FIRST_SEQUENCE = 0;
+
+    private final ApplicantRepository applicantRepository;
+    private final ProcessRepository processRepository;
+    private final DashboardRepository dashboardRepository;
+    private final EvaluationRepository evaluationRepository;
+
+    public ProcessesResponse findByDashboardId(long dashboardId) {
+        boolean dashboardExists = dashboardRepository.existsById(dashboardId);
+        if (!dashboardExists) {
+            throw new DashboardNotFoundException();
+        }
+
+        return new ProcessesResponse(processRepository.findAllByDashboardId(dashboardId)
+                .stream()
+                .map(this::toProcessResponse)
+                .toList());
+    }
+
+    private ProcessResponse toProcessResponse(Process process) {
+        List<DashboardApplicantResponse> dashboardApplicantResponses = createDashboardApplicantResponses(process);
+
+        return new ProcessResponse(
+                process.getId(),
+                process.getSequence(),
+                process.getName(),
+                process.getDescription(),
+                dashboardApplicantResponses
+        );
+    }
+
+    private List<DashboardApplicantResponse> createDashboardApplicantResponses(Process process) {
+        return applicantRepository.findAllByProcess(process)
+                .stream()
+                .map(applicant -> toDashboardApplicantResponse(applicant, process))
+                .toList();
+    }
+
+    private DashboardApplicantResponse toDashboardApplicantResponse(Applicant applicant, Process process) {
+        int evaluationCount = evaluationRepository.countByApplicantAndProcess(applicant, process);
+        return new DashboardApplicantResponse(
+                applicant.getId(),
+                applicant.getName(),
+                applicant.getCreatedDate(),
+                applicant.getIsRejected(),
+                evaluationCount
+        );
+    }
+
+    @Transactional
+    public void create(long dashboardId, ProcessCreateRequest request) {
+        List<Process> allByDashboardId = processRepository.findAllByDashboardId(dashboardId);
+        validateProcessCount(allByDashboardId);
+
+        allByDashboardId.stream()
+                .filter(process -> process.getSequence() >= request.sequence())
+                .forEach(Process::increaseSequenceNumber);
+
+        processRepository.save(new Process(
+                request.sequence(),
+                request.name(),
+                request.description(),
+                dashboardRepository.findByIdOrThrow(dashboardId))
+        );
+    }
+
+    private void validateProcessCount(List<Process> processes) {
+        if (processes.size() == MAX_PROCESS_COUNT) {
+            throw new ProcessBadRequestException();
```

Comment:
```suggestion
            throw new ProcessBadRequestException("최대 프로세스 수를 초과하여 더 이상 프로세스를 추가할 수 없습니다.");
```
현재 BadRequest 메시지가 `잘못된 프로세스 관련 요청입니다.`인데, 이는 너무 불명확하다고 생각합니다. BadRequest 관련한 메시지는 이 부분에 String으로 넣는 것에 대한 의견이 궁금합니다.

Reply:
메시지가 불명확하면 생기는 문제가 무엇인가요? 어떤 니즈 때문에 더욱 명확하게 작성하고 싶으신지를 알아야할 것 같아서요.

제 의견은 `에러 메시지가 많아지는 상황에서 메시지가 불명확하면, 에러를 구분하고 디버깅하기 어렵다`인데, 명오는 저랑 의견이 같으실까요?

Reply:
러쉬 의견쪽입니다.
클라이언트가 요청을 보냈을 때, 왜 잘못된 것인지 파악할 수 있어야 한다고 생각합니다.